src/main/webapp/app/entities/club-family/family-member.tsx

```diff
@@ -0,0 +1,131 @@
+
+import AuthorizationChecker from 'app/shared/components/authorization-checker/authorization-checker';
+import CCRole from 'app/shared/model/enum/cc-role.enum';
+import EventRole from 'app/shared/model/enum/event-role.enum';
+import MemberCard from './member-card';
+import FilterSearchBar from 'app/shared/components/advancedSearchModal/advancedSearchModal';
+import EventModal from 'app/shared/components/eventModal/event-modal';
+
+import { getUsersWithFilter, reset as resetUsers } from 'app/entities/user-cc-info/user-cc-info.reducer';
+import { getClubFamilyDetails } from 'app/shared/services/club-family-info.service';
+import { reset as resetFilter } from 'app/shared/components/advancedSearchModal/advancedSearchModal.reducer';
+
+export interface IFamilyMemberProps extends StateProps, DispatchProps, RouteComponentProps<{ familyCode: string }> {}
+
+export interface IFamilyMemberState {
+  searchModalIsOpen: boolean;
+}
+class FamilyMember extends React.Component<IFamilyMemberProps, IFamilyMemberState> {
+  constructor(props: IFamilyMemberProps) {
+    super(props);
+    this.state = {
+      searchModalIsOpen: false,
+    };
+  }
+
+  async componentDidMount() {
+    this.props.getUsersWithFilter(this.props.match.params.familyCode);
+  }
+
+  componentWillUnmount() {
+    this.props.resetFilter();
+    this.props.resetUsers();
+  }
+
+  showSearchModal = (): void => {
+    this.setState({
+      searchModalIsOpen: true,
+    });
+  };
+
+  toggleSearchModal = (): void => {
+    this.setState({
+      searchModalIsOpen: !this.state.searchModalIsOpen,
+    });
+  };
+
+  onEditButtonClick = (userCCInfoId?: number): void => {};
+
+  render() {
+    const { users, match } = this.props;
+    const { searchModalIsOpen } = this.state;
+    const familyName = getClubFamilyDetails(match.params.familyCode).name;
+    return (
+      <div>
+        <FilterSearchBar
+          isOpen={searchModalIsOpen}
+          toggleModal={this.toggleSearchModal}
+          familyCode={this.props.match.params.familyCode}
+          searchUsers={this.props.getUsersWithFilter}
+        />
+        <h2 id="event-activity-heading" className="event-module-heading">
+          {familyName ? translate(familyName) : null}
+        </h2>
+        <div className="mx-4">
+          <AuthorizationChecker ccRole={CCRole.ADMIN}>
+            <Link
+              className="btn btn-action jh-create-entity w-100 my-2"
+              to={{ pathname: `/entity/members/cc-family/${match.params.familyCode}/new`, state: { from: this.props.match.url } }}
+            >
+              <Translate contentKey="entity.action.add">Add</Translate>
+            </Link>
+          </AuthorizationChecker>
+          <InputGroup>
+            <InputGroupAddon className="search-bar-prepend" addonType="prepend" color="white">
+              {/* ADD IN SEARCH ONCLICK TO THE BUTTON BELOW*/}
+              <Button color="primary">
+                <FontAwesomeIcon icon="search" />
+              </Button>
```

Comment:
This is TODO item after merge with #136 ?

Reply:
Yes. Currently the search bar has no functionality. Will fix this after merge with #136.